src/functions-reference/complex_matrix_operations.Rmd

```diff
@@ -453,5 +692,406 @@ The sum of the values in x, or 0 if x is empty
+\index{{\tt \bfseries append\_col }!{\tt (complex\_matrix x, complex\_vector y): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`append_col`**`(complex_matrix x, complex_vector y)`<br>\newline
+Combine matrix x and vector y by columns. The matrix and the vector must have
+the same number of rows.
+`r since("2.30")`
+
+<!-- complex_matrix; append_col; (complex_vector x, complex_matrix y); -->
+\index{{\tt \bfseries append\_col }!{\tt (complex\_vector x, complex\_matrix y): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`append_col`**`(complex_vector x, complex_matrix y)`<br>\newline
+Combine vector x and matrix y by columns. The vector and the matrix must have
+the same number of rows.
+`r since("2.30")`
+
+<!-- complex_matrix; append_col; (complex_vector x, complex_vector y); -->
+\index{{\tt \bfseries append\_col }!{\tt (complex\_vector x, complex\_vector y): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`append_col`**`(complex_vector x, complex_vector y)`<br>\newline
+Combine vectors x and y by columns. The vectors must have the same number of
+rows.
+`r since("2.30")`
+
+<!-- complex_row_vector; append_col; (complex_row_vector x, complex_row_vector y); -->
+\index{{\tt \bfseries append\_col }!{\tt (complex\_row\_vector x, complex\_row\_vector y): complex\_row\_vector}|hyperpage}
+
+`complex_row_vector` **`append_col`**`(complex_row_vector x, complex_row_vector y)`<br>\newline
+Combine row vectors x and y of any size into another row vector.
+`r since("2.30")`
+
+<!-- complex_row_vector; append_col; (complex x, complex_row_vector y); -->
+\index{{\tt \bfseries append\_col }!{\tt (complex x, complex\_row\_vector y): complex\_row\_vector}|hyperpage}
+
+`complex_row_vector` **`append_col`**`(complex x, complex_row_vector y)`<br>\newline
+Append x to the front of y, returning another row vector.
+`r since("2.30")`
+
+<!-- complex_row_vector; append_col; (complex_row_vector x, complex y); -->
+\index{{\tt \bfseries append\_col }!{\tt (complex\_row\_vector x, complex y): complex\_row\_vector}|hyperpage}
+
+`complex_row_vector` **`append_col`**`(complex_row_vector x, complex y)`<br>\newline
+Append y to the end of x, returning another row vector.
+`r since("2.30")`
+
+### Vertical concatenation
+
+<!-- complex_matrix; append_row; (complex_matrix x, complex_matrix y); -->
+\index{{\tt \bfseries append\_row }!{\tt (complex\_matrix x, complex\_matrix y): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`append_row`**`(complex_matrix x, complex_matrix y)`<br>\newline
+Combine matrices x and y by rows. The matrices must have the same number of
+columns.
+`r since("2.30")`
+
+<!-- complex_matrix; append_row; (complex_matrix x, complex_row_vector y); -->
+\index{{\tt \bfseries append\_row }!{\tt (complex\_matrix x, complex\_row\_vector y): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`append_row`**`(complex_matrix x, complex_row_vector y)`<br>\newline
+Combine matrix x and row vector y by rows. The matrix and the row vector must
```

Comment:
same---"by row"

Reply:
Many of these wording suggestions would apply to the real-valued versions of these functions (I mostly copy/pasted from there). Would you like me to update the wording in both places?